loops/haxe/Code.hx

```diff
@@ -0,0 +1,23 @@
+import haxe.ds.Vector;
+
+class Code {
+	public static function main() {
+		var u:Int = Std.parseInt(Sys.args()[0]); // Command line input number
+		var r:Int = Std.random(10000);
+		var a = new Vector<Int>(10000); // Initialize a vector with 10,000 elements
+
+		for (i in 0...10000) { // Outer loop with 10k iterations
+			#if !java
+			a[i] = 0; // Set all VectorData to 0, this part is unnecessary since the JVM target already fills the values with 0.
```

Comment:
The zeroing out needs to be done before the loop to match the other test cases, so this should be removed / changed.

Reply:
https://github.com/bddicken/languages/pull/70/commits/0b76083104c507040c01c80e17e30cf570b3dd4d

Fixed